lib/spritesheet.js

```diff
@@ -83,11 +84,17 @@ module.exports = function Spritesheet(name, scaledSprites, groupConfig, config,
         that.outputFilesize = data.outputFilesize;
     }
 
+     function calculateHash() {
+        var hash = crypto.createHash("sha1");
+        hash.update(_.pluck(scaledSprites, "path").sort().join(" ") + "_");
+        hash.update(that.extension + "_");
+        hash.update((groupConfig.quality ? groupConfig.quality : "noquality") + "_");
+        hash.update((groupConfig.compression ? groupConfig.compression : "nocompression"));
+        return name + "_" + hash.digest("hex");
+    }
+
```

Comment:
could you add a unit test for this? it's quite hard to understand this code without understanding what the input values are, and that's not easy to find